dali/pipeline/operators/displacement/displacement_filter_impl_cpu.h

```diff
@@ -110,6 +112,67 @@ class DisplacementFilter<CPUBackend, Displacement,
   USE_OPERATOR_MEMBERS();
 
  private:
+  // TODO(klecki) We could probably interpolate with something other than float,
+  // for example integer type
+  struct LinearCoefs {
+    LinearCoefs(float rx, float ry) : rx(rx), ry(ry), mrx(1.f - rx), mry(1.f - ry) {}\
+    LinearCoefs() = default;
+    float rx, ry, mrx, mry;
+  };
+
+  template <typename T>
+  bool ShouldTransform(Point<T> p) {
+    return p.x >= 0 && p.y >= 0;
+  }
+
+  template <typename T>
+  Index PointToInIdx(Point<T> p, Index H, Index W, Index C) {
```

Comment:
`p` could be a `const &`. Same thing applies for few function definitions below

Reply:
As above, I would prefer value semantics over const&.